frontend/gateway/grpcclient/client.go

```diff
@@ -51,11 +50,11 @@ func New(ctx context.Context, opts map[string]string, session, product string, c
 	}
 
 	if resp.FrontendAPICaps == nil {
-		resp.FrontendAPICaps = defaultCaps()
+		resp.FrontendAPICaps = util.PointerSlice(defaultCaps())
```

Comment:
If it doesn't change anything else we should consider changing definition of `defaultCaps()` etc in these cases instead of converting.

Reply:
Y

Reply:
This PointerSlice and some other type-casting functions are gone at fc4c2fe3c879d084f42639c5dd6b28d0e9f87c48.